app/src/main/java/org/p2p/wallet/send/ui/main/SendPresenter.kt

```diff
@@ -391,15 +389,15 @@ class SendPresenter(
         checkAddress(result.addressState.address)
     }
 
-    private fun handleAddressOnlyResult(result: SearchResult.AddressOnly) {
+    private fun handleAddressOnlyResult(result: SearchResult.AddressFound) {
         view?.showAddressOnlyTarget(result.addressState.address)
         checkAddress(result.addressState.address)
     }
 
-    private fun handleWrongResult(result: SearchResult.InvalidAddress) {
-        view?.showWrongAddressTarget(result.addressState.address.cutEnd())
-        view?.hideAccountFeeView()
-    }
+//    private fun handleWrongResult(result: SearchResult.InvalidAddress) {
```

Comment:
comments are ok?

Reply:
it will be removed. I added the `deprecated` annotation